client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEdit.tsx

```diff
@@ -32,28 +32,44 @@ import {
   getAllocatedQuantity,
   getAllocatedPacks,
 } from './utils';
-import { useOutbound } from '../../api';
+import { DraftItem, useOutbound } from '../../api';
 import { DraftOutboundLine } from '../../../types';
 
 interface ItemDetailsModalProps {
   isOpen: boolean;
   onClose: () => void;
-  item: ItemRowFragment | null;
+  draft: DraftItem | null;
   mode: ModalMode | null;
 }
 
+const useFocusNumberOfPacksInput = (draft: DraftItem | null) => {
+  const batch = draft?.barcode?.batch;
+
+  useEffect(() => {
+    setTimeout(() => {
+      if (!batch) return;
+      const input = document.getElementById(`pack_quantity_${batch}`);
+      if (input) {
+        input.focus();
+      }
+    }, 500);
```

Comment:
Why would this be 500 ? Is this estimated time that it takes for modal to load ? Wouldn't it be different based on hardware or number of lines ?

Reply:
possibly, yeah. It appeared to just require a timeout to allow the focus to work, perhaps the rendering time would impact this, though we shouldn't have that many lines in this table anyway. If the input doesn't focus it isn't the end of the world, just not as nice to use